cmd/checkmetrics/compare.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"errors"
+	"fmt"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// metricsCheck is a placeholder struct for us to attach the methods to and make
+// it clear they belong this grouping. Maybe there is a better way?
+type metricsCheck struct {
+}
+
+// titleFormat contains the format string for the table header
+// We lay out two format strings so we can align both the title and the
+// lines of the report to try and make it 'tabulised'
+//                        P/F: Name: Min   < Mean  < Max:  Iter: SD:   CoV %"
+var titleFormat string = "%4s: %20s: %7s < %7s < %7s: %5s: %7s: %7s %% "
```

Comment:
I did wonder if there was a suitable package as I was writing it (you know how it goes - first was just one little whitespace alignment to make it nice... which then grows into a whole table format).
I'll go have a look at those options. One thing we may end up wanting is for #104 reporting tool to be able to parse the output of checkmetrics - in which case we may also add some arguments to output in a 'standard format'.